db/migrate/20240216095511_create_cohorts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateCohorts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :cohorts do |t|
+      t.integer :start_year, null: false
+      t.datetime :registration_start_date
```

Comment:
maybe this should also be `null: false`? if we plan to populate this from previous cohorts 🤔 let me know your thoughts